src/ui/pages/SwapForm/Quotes/useQuotes.ts

```diff
@@ -92,12 +98,18 @@ export function useQuotes({
         value,
         getDecimals({ asset: position.asset, chain })
       ).toFixed();
+
+      const { default: defaultSlippagePercent } = getSlippageOptions(chain);
+      const slippagePercent = String(
+        userSlippage ? toPercents(userSlippage) : defaultSlippagePercent
```

Comment:
We shouldn't forget this PR top be merged https://github.com/zeriontech/transactions-ui/pull/15/files and package.json to be updated

Reply:
Also here we need to use `??` instead of `?`
Slippage: 0 technically should be possible.

Reply:
`userSlippage == null ? toPercents(userSlippage) : defaultSlippagePercent`